community-operators/knative-kafka-operator/knative-kafka-operator.v0.6.0.clusterserviceversion-v0.6.0.yaml

```diff
@@ -0,0 +1,173 @@
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: knative-kafka-operator
+            spec:
+              containers:
+              - command:
+                - knative-kafka-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-kafka-operator
+                image: quay.io/openshift-knative/knative-kafka-operator:v0.6.0
+                imagePullPolicy: Always
+                name: knative-kafka-operator
+                resources: {}
+              serviceAccountName: knative-kafka-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - knative-kafka-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - serving.knative.dev
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - eventing.knative.dev
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: knative-kafka-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - serverless
+  - eventing
+  - kafka
+  links:
+  - name: Knative Eventing sources
+    url: https://knative.dev/docs/eventing/sources/
+  - name: Knative Eventing channels
+    url: https://knative.dev/docs/eventing/channels/
+  - name: Website
+    url: https://www.knative.dev/docs/eventing/
+  - name: Documentation
+    url: https://www.knative.dev/docs/
+  maintainers:
+  - email: knative@redhat.com
+    name: Knative team
+  maturity: alpha
+  provider:
+    name: Knative Community
```

Comment:
This should be "Red Hat", I believe. Whenever we publish an upstream-approved version, we'll use the "Knative Community" name then.

Reply:
Thanks, Jim! Fixed.